aces/idt/core/common.py

```diff
@@ -47,14 +54,47 @@
     "clf_processing_elements",
 ]
 
-SDS_COLORCHECKER_CLASSIC = tuple(SDS_COLOURCHECKERS["ISO 17321-1"].values())
+
+def get_sds_colour_checker(colour_checker_name):
+    """
+    Get the Reference reflectances for the given colour checker
+
+    Parameters
+    ----------
+    colour_checker_name: The name of the colour checker name
+
+    Returns
+    -------
+        SDS_COLORCHECKER : tuple
+
+    """
+    return tuple(SDS_COLOURCHECKERS[colour_checker_name].values())
+
+
+def get_sds_illuminant(illuminant_name):
+    """
+    Get *ACES* reference illuminant spectral distribution, for the given illuminant name
+
+    Parameters
+    ----------
+    illuminant_name: The name of the illuminant
+
+    Returns
+    -------
+        SpectralDistribution
+
+    """
+    return SDS_ILLUMINANTS[illuminant_name]
```

Comment:
**Minor**

Isn't that a bit "busy" for what a very simple lookups? They don't seem to be used a lot either.

Reply:
So looking at this, the only reason these seem to exist is because in common.py we have these constants

SDS_COLORCHECKER_CLASSIC
RGB_COLORCHECKER_CLASSIC_ACES
SD_ILLUMINANT_ACES

These are not used anywhere else in the project and the

generate_reference_colour_checker takes them as default options on the input.

If we remove the default options to generate_reference_colour_checker

the values are already defaulted in the projectSettings, so we could then just loose all of this and move just the

SDS_ILLUMINANTS[illuminant_name] & tuple(SDS_COLOURCHECKERS[colour_checker_name].values())
into the project settings which it then passes into the generate_reference_colour_checker.

This removes 3 unused constants, and 2 un needed functions into a single function within the project_settings